yarn-project/end-to-end/src/e2e_cross_chain_messaging.test.ts

```diff
@@ -245,11 +246,16 @@ describe('e2e_cross_chain_messaging', () => {
       secretHashForL2MessageConsumption,
     );
 
+    // get message leaf index, needed for claiming in public
+    const maybeIndexAndPath = await aztecNode.getL1ToL2MessageMembershipWitness('latest', msgHash, 0n);
```

Comment:
Maybe we can add a couple of new oracles including one to just get the leaf index. Probably not necessary in this PR though (if it is even the right move). I think we also want one to do something like "checkMessageExists(message, index)" for use in the journal.

Reply:
Discussed. Happens in child PR.